torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -159,6 +198,10 @@ std::string valueName(Value * n) {
 }
 
 const char * scalarTypeName(at::ScalarType type) {
+  if (type == at::ScalarType::Half) {
+    return "half";
+  }
```

Comment:
Maybe would be even clearer if we switch the macro below from `AT_FORALL_SCALAR_TYPES` to `AT_FORALL_SCALAR_TYPES_EXCEPT_HALF`.

Reply:
That would be clearer; give me a moment and I'll fix that up.